### ISSUE TITLE
Failure to "Transfer"

### ISSUE THREAD

**Issue by Zageron** [librespot-org/librespot-java#288]
**Describe the bug**
- Last night I was listening to a podcast on my phone connected to librespot.
- This evening I tried to transfer my music from my PC to librespot.
- Nothing happens.

**To Reproduce**
- Reboot system.
- Transfer an active session.

**Expected behavior**
- Music playing on PC should begin playing on librespot.

**Screenshots/Stracktraces/Logs**
```
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,324 DEBUG Player:293 - Received command: Transfer
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,346 DEBUG Player:365 - Loading context (transfer), uri: spotify:station:album:0jMdvHngDq93H361gAh9aS
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,370 TRACE StateWrapper:1084 - Initialized current track index to 24.
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,382 DEBUG Player:225 - Loading session, id: CsUGvdHJ70rYs5/ySohedA, play: true
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,384 TRACE PlayerQueue:141 - Queue has been cleared.
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,385 INFO  PlayerSession:53 - Created new session. {id: CsUGvdHJ70rYs5/ySohedA}
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,391 TRACE MercuryClient:140 - Send Mercury request, seq: 291, uri: hm://event-service/v1/events, method: POST
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,394 DEBUG Player:253 - Loading track, id: null, session: CsUGvdHJ70rYs5/ySohedA, playback: null, play: true
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,396 ERROR DealerClient:112 - Failed handling request. {key: 11492131-c5d7-421f-a23f-b08a9234ccc6}
Jan 04 05:55:45 spotipi java[597]: java.lang.IllegalStateException: null
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.StateWrapper.getCurrentPlayableOrThrow(StateWrapper.java:558) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.Player.loadTrack(Player.java:254) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.Player.loadSession(Player.java:239) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.Player.handleTransferState(Player.java:370) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.Player.command(Player.java:300) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.state.DeviceStateHandler.notifyCommand(DeviceStateHandler.java:110) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.player.state.DeviceStateHandler.onRequest(DeviceStateHandler.java:176) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.dealer.DealerClient.lambda$handleRequest$1(DealerClient.java:107) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.common.AsyncWorker.lambda$new$0(AsyncWorker.java:20) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at xyz.gianlu.librespot.common.AsyncProcessor.lambda$submit$1(AsyncProcessor.java:40) ~[librespot-java-api-v1.5.3.jar:1.5.3]
Jan 04 05:55:45 spotipi java[597]:         at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
Jan 04 05:55:45 spotipi java[597]:         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
Jan 04 05:55:45 spotipi java[597]:         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
Jan 04 05:55:45 spotipi java[597]:         at java.lang.Thread.run(Thread.java:834) [?:?]
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,523 TRACE MercuryClient:183 - Handling packet, cmd: MercuryReq, seq: 291, flags: 1, parts: 1
Jan 04 05:55:45 spotipi java[597]: 2021-01-04 05:55:45,523 DEBUG EventService:35 - Event sent. {body: 557|3|CsUGvdHJ70rYs5/ySohedA|spotify:station:album:0jMdvHngDq93H361gAh9aS|spotify:station:album:0jMdvHngDq93H361gAh9aS|1609739746315||48|, result: 200}
```

**Version/Commit**
1.5.3

**Comment by Zageron** [comment]
Looks like this should be resolved in 1.5.4.

**Comment by devgianlu** [comment]
> Looks like this should be resolved in 1.5.4?

Yep, duplicate of #281.
